src/terminal/Line.cpp

```diff
@@ -188,7 +189,7 @@ InflatedLineBuffer<Cell> inflate(TriviallyStyledLineBuffer const& input)
     }
 
     while (columns.size() < unbox<size_t>(input.displayWidth))
-        columns.emplace_back(Cell { input.attributes });
+        columns.emplace_back(Cell { GraphicsAttributes {} });
```

Comment:
That's been on my list since some time already but never had time recently. Thanks. :+1: :)